src/main/java/com/jland/web/controller/PresentationController.java

```diff
@@ -0,0 +1,35 @@
+package com.jland.web.controller;
+
+import com.jland.service.PresentationService;
+import com.jland.web.dto.PresentationDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+
+@Controller
+@RequestMapping(value = "/presentation", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+public class PresentationController {
+    private final Logger LOGGER = LoggerFactory.getLogger(getClass());
```

Comment:
usually private static final and after it LOGGER is nice

Reply:
You want to replace getClass() with PresentationController.class ?

Reply:
yes. And we will have common approach for LOGGER. I mean class with static and non-static method